src/events/eventToAction.js

```diff
@@ -0,0 +1,119 @@
+import {
+  EVENT_NEW_MESSAGE,
+  EVENT_PRESENCE,
+  EVENT_REACTION_ADD,
+  EVENT_REACTION_REMOVE,
+  EVENT_STREAM_ADD,
+  EVENT_STREAM_REMOVE,
+  EVENT_STREAM_UPDATE,
+  EVENT_SUBSCRIPTION_ADD,
+  EVENT_SUBSCRIPTION_REMOVE,
+  EVENT_SUBSCRIPTION_UPDATE,
+  EVENT_SUBSCRIPTION_PEER_ADD,
+  EVENT_UPDATE_MESSAGE,
+  EVENT_UPDATE_MESSAGE_FLAGS,
+  EVENT_USER_ADD,
+  EVENT_USER_REMOVE,
+  EVENT_USER_UPDATE,
+} from '../constants';
+
+import { getUserById } from '../users/usersSelectors';
+
+const opToActionSubscription = {
```

Comment:
1. I agree partially, but also it is much better than before. It turns out a lot of events have 4-5 operations. It just becomes too much 'ifs' and repetitive code for no good reason.
2. Yeah, some operations do indeed have different params. I am using `...event` to keep it simple.